spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb

```diff
@@ -120,6 +122,43 @@
     end
   end
 
+  describe '#partner_agency' do
+    subject(:partner_agency) { presenter.partner_agency }
+
+    it 'returns partner agency' do
+      expect(partner_agency).to eq('http://localhost:3000')
+    end
+  end
+
+  describe '#days_reminder' do
+    subject(:days_reminder) { presenter.days_reminder }
+
+    context '4 days until due date' do
+
+      before do
+        allow(presenter).to receive(:enrollment.days_to_due_date).and_return(4)
```

Comment:
I think this is a typo
```suggestion
        allow(presenter).to receive(enrollment.days_to_due_date).and_return(4)
```